cinema/tests/test_movie.py

```diff
@@ -0,0 +1,51 @@
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework import status
+from django.contrib.auth import get_user_model
+from rest_framework.test import APIClient
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+from cinema.tests.test_movie_api import MOVIE_URL
+
+MOVIE_URL = reverse("cinema:movie-list")
```

Comment:
The `MOVIE_URL` is defined twice: once in line 8 and again in line 10. This redundancy can lead to confusion or errors if the URL changes in one place but not the other. Consider removing one of these definitions to maintain consistency.